package.json

```diff
@@ -47,14 +47,17 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.12.1",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.5",
-    "onfido-sdk-ui": "^6.3.1",
+    "onfido-sdk-ui": "onfido/onfido-sdk-ui#exp/typescript",
```

Comment:
This should be updated with the latest release containing TypeScript implementation